flows/coordinatematch/wcs.py

```diff
@@ -0,0 +1,214 @@
+from copy import deepcopy
+
+import numpy as np
+import astropy.wcs
+
+from scipy.optimize import minimize
+from scipy.spatial.transform import Rotation
+
+class WCS () :
```

Comment:
Git this another name to avoid clashes with astropy.wcs.WCS

Reply:
> Why is it needed to have the API URL in the config file... This sounds to me like a way to open a security hole... I change that config to point to something nasty and bad things happen... And it will break existing setups if it is not defined

I Don't understand not having it. We just have to specify this in the readme. Currently this is defined in the python code which would be trivial to change also. I don't see the problem with having it as a config, in fact, I think that's the preferred approach so that we can divorce the pipeline from the specific website in the (distant) future and release it as a package.